index.html

```diff
@@ -189,14 +189,22 @@ <h2 class="font-work-sans font-semibold mb-5 text-3xl">
                 <p class="mt-2 underline">Youtube</p>
             </a>
 
-            <a href="" target="_blank" class="hover:text-white inline-block">
+            <a href="" target="_blank" class="hover:text-white inline-block mr-4">
                 <img
                     src="resources/assets/website.png"
                     alt="jslovers twitter website"
                     class="hover:invert mx-auto"
                 />
                 <p class="mt-2 underline">Website</p>
             </a>
+            <a href="https://discord.com/invite/89QtcRw" target="_blank" class="hover:text-white inline-block">
+                <img
+                    src="resources/assets/discord.png"
+                    alt="jslovers twitter website"
```

Comment:
Can you please keep the alt tag empty here? alt=""

Reply:
Removed alt tag